modules/openshift-storage-common-terms.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * storage/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-storage-common-terms_{context}"]
+= Common terms
+
+This glossary defines common terms that are used in the architecture content. These terms help you understand {product-title} architecture effectively.
+
+Access modes:: Volume access modes describe volume capabilities. You can use access modes to match persistent volume claim (PVC) and persistent volume (PV). The following are the examples of access modes:
+
+* ReadWriteOnce (RWO)
+* ReadOnlyMany (ROX)
+* ReadWriteMany (RWX)
+* ReadWriteOncePod (RWOP)
+
+Cinder:: The Block Storage service for {rh-openstack-first} which manages the administration, security, and scheduling of all volumes.
+
+Config map:: A config map provides a way to inject configuration data into pods. You can reference the data stored in a config map in a volume of type `ConfigMap`. Applications running in a pod can consume this data.
+
+Container Storage Interface (CSI)::
+An API specification for the management of container storage across different container orchestration (CO) systems.
+
+Dynamic Provisioning::
+The framework allows you to create storage volumes on-demand, eliminating the need for cluster administrators to pre-provision persistent storage.
+
+Ephemeral storage::
+Pods and containers can require temporary or transient local storage for their operation. The lifetime of this ephemeral storage does not extend beyond the life of the individual pod, and this ephemeral storage cannot be shared across pods.
+
+External CSI Controllers::
```

Comment:
I think we don't need this item, and the explanation is not accurate.

Reply:
Addressed